test/system/src/main/java/io/pravega/test/system/framework/kubernetes/K8sClient.java

```diff
@@ -664,25 +664,25 @@ public void deletePVC(String namespace, String name) {
         AtomicBoolean shouldRetry = new AtomicBoolean(true);
 
         return Futures.loop(shouldRetry::get,
-                            () -> Futures.delayedFuture(Duration.ofSeconds(5), executor) // wait for 5 seconds before checking for status.
-                                    .thenCompose(v -> getStatusOfPodWithLabel(namespace, labelName, labelValue)) // fetch status of pods with the given label.
-                                    .thenApply(podStatuses -> podStatuses.stream()
-                                            // check for pods where all containers are running.
-                                            .filter(podStatus -> {
-                                                if (podStatus.getContainerStatuses() == null) {
-                                                    return false;
-                                                } else {
-                                                    return podStatus.getContainerStatuses()
-                                                            .stream()
-                                                            .allMatch(st -> st.getState().getRunning() != null);
-                                                }
-                                            }).count()),
-                            runCount -> { // Number of pods which are running
-                                log.debug("Expected running pod count : {}, actual running pod count :{}.", expectedPodCount, runCount);
-                                if (runCount == expectedPodCount) {
-                                    shouldRetry.set(false);
-                                }
-                            }, executor);
+                () -> Futures.delayedFuture(Duration.ofSeconds(5), executor) // wait for 5 seconds before checking for status.
```

Comment:
this seems like formatting only change and below. Could you undo this?

Reply:
This is similar in master also. I compared my code against master and no changes are there.